store/store.go

```diff
@@ -531,7 +531,12 @@ func (s *Store) newRequest(reqOptions *requestOptions, user *auth.UserState) (*h
 		if err != nil {
 			return nil, err
 		}
-		// TODO: if device session is empty, request new session
+		if device.SessionMacaroon == "" {
+			err = s.refreshDeviceSession()
```

Comment:
shouldn't we simply ignore for now if the error is state.ErrNoState? and log and continue? instead of keeping this blocked